src/main/java/fr/foodlimit/api/place/PlaceService.java

```diff
@@ -0,0 +1,85 @@
+package fr.foodlimit.api.place;
+
+import fr.foodlimit.api.food.FoodRepository;
+import fr.foodlimit.api.shared.models.Food;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'fr.foodlimit.api.shared.models.Food'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)